modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateUpdater.scala

```diff
@@ -338,6 +346,27 @@ object ConsensusStateUpdater {
     private def pickMajority[A: Order](proposals: List[A]): Option[A] =
       proposals.foldMap(a => Map(a -> 1)).toList.map(_.swap).maximumOption.map(_._2)
 
+    private def pickValidatedMajority(
+      lastSignedArtifact: Signed[Artifact],
+      trigger: ConsensusTrigger,
+      resources: ConsensusResources[Artifact]
+    )(proposals: List[Hash]): F[Option[Hash]] =
+      proposals
+        .foldMap(a => Map(a -> 1))
+        .toList
+        .map(_.swap)
+        .sortBy(_._1)
+        .foldLeftM(none[Hash]) {
+          case (None, (_, majorityHash)) =>
```

Comment:
Before `foldLeftM` Please map all items to `hash` instead of `(count, hash)` since `count` is unused